acj/static/test/config/protractor_cucumber.js

```diff
@@ -1,6 +1,6 @@
 exports.config = {
     //seleniumAddress: 'http://localhost:4444/wd/hub',
-	seleniumServerJar: '../../../../node_modules/protractor/selenium/selenium-server-standalone-2.47.1.jar',
+	//seleniumServerJar: '../../../../node_modules/protractor/selenium/selenium-server-standalone-2.47.1.jar',
```

Comment:
The jar file no longer exists in the location referenced in the newer version. Protractor seems to auto load it itself.